README.md

```diff
@@ -166,7 +166,7 @@ You write extra params with rules with @LRDparam in the comment line as one line
      * // either space or pipe
      * @LRDparam nickaname string|nullable|max:32
      * // override the default response codes
-     * @LRDparam responses 200,422
+     * @LRDparam responses 200|422
```

Comment:
This is actually wrong. It should be `@LRDresponses`, not `@LRDparam`

Reply:
I'll make a pull request and update the readme with the changes from https://github.com/rakutentech/laravel-request-docs/pull/314 explained too

Reply:
here you go: https://github.com/rakutentech/laravel-request-docs/pull/318